pcsx2-qt/MainWindow.cpp

```diff
@@ -2905,6 +2910,35 @@ void MainWindow::goToWikiPage(const GameList::Entry* entry)
 	QtUtils::OpenURL(this, fmt::format("https://wiki.pcsx2.net/{}", entry->serial).c_str());
 }
 
+void MainWindow::openScreenshotsFolderForGame(const GameList::Entry* entry)
+{
+    std::string base_path = EmuFolders::Snapshots;
+
+	std::string game_name;
+	if (std::string name = entry->title; !name.empty())
+	{
+		game_name = name;
+		Path::SanitizeFileName(&game_name);
+		if (game_name.length() > 219)
```

Comment:
why 219?

Reply:
Sure, but that code also appends the serial, a timestamp, and potentially a counter to the game title. It seems it's trying to keep the total file name length below 250.

In this case you're not appending anything else to the title.